src/msw/uilocale.cpp

```diff
@@ -197,6 +407,62 @@ class wxUILocaleImplName : public wxUILocaleImpl
         return s_canUse == 1;
     }
 
+    static wxArrayString GetPreferredUILanguages()
+    {
+        wxArrayString preferred;
+
+        if (CanUse())
+        {
+            ULONG numberOfLanguages = 0;
+            ULONG bufferSize = 0;
+            if (ms_GetUserPreferredUILanguages(MUI_LANGUAGE_NAME, &numberOfLanguages, NULL, &bufferSize))
+            {
+                wxScopedArray<WCHAR> languages(bufferSize);
+                if (ms_GetUserPreferredUILanguages(MUI_LANGUAGE_NAME, &numberOfLanguages, languages.get(), &bufferSize))
```

Comment:
We probably should at least log the error here as well if it happens.

Reply:
Yes, I'll add that, although it is unlikely to fail on the second call.